presto-pinot-toolkit/src/test/java/com/facebook/presto/pinot/TestPinotBrokerPageSourceSql.java

```diff
@@ -0,0 +1,199 @@
+                        ImmutableList.of(varchar("event_id"), varchar("event_name"), bigint("event_time"), groupCity, groupCountry, bigint("group_id"), varchar("group_name"), bigint("mtime"), integer("rsvp_count"), varchar("venue_name")),
+                        ImmutableList.of(0, 1, 2, 3, 4, 5, 6, 7, 8, 9),
+                        Optional.empty()}
+        };
+    }
+
+    @Test(dataProvider = "sqlResponses")
+    public void testPopulateFromSql(
+            String sql,
+            String sqlResponse,
+            List<PinotColumnHandle> actualHandles,
+            List<Integer> expectedColumnIndices,
```

Comment:
Is `expectedColumnIndices` still relevant for testing the SQL end point ?

Reply:
Yes. I wanna keep this for a while as I haven't changed the query like `SELECT  DISTINCT a` which is currently using hidden column to rewrite the query.

Reply:
I see. So am I reading the test on line 103 of this file: "SELECT distinct group_country FROM meetupRsvp  LIMIT 1000000" correctly ? It uses expectedColumnIndices to do this logic ?

Reply:
I think here is another issue that `SELECT distinct group_country FROM meetupRsvp LIMIT 1000000` is not pushed down from recent code.
This is always translated to INSTANCE query `SELECT group_country FROM meetupRsvp LIMIT X`.
I can address this hidden column in the next PR.